configs/templates/internal_options.txt

```diff
@@ -224,6 +224,7 @@ PREPARE_IMAGE_NAME = None
 NEST_CONTAINERS_ENABLED = $False
 NEST_CONTAINERS_REPOSITORY = ibmcb
 NEST_CONTAINERS_INSECURE_REGISTRY = $False
+CONFIGURE_FIREWALL = $False
```

Comment:
Could we make the true by default? I kind of want to be forced to experience the pain while we practice packaging CloudBench......... by making this true by default, it will also help others gain more confidence and trust (including SPEC).

Reply:
Very good point... will do.